resources/views/email/notification.blade.php

```diff
@@ -77,4 +77,24 @@
                             </li>
                         @endforeach
                         </ul>
+
+                      </td>
+                    </tr>
+                  </table>
+                </td>
+              </tr>
+
+                <!-- END MAIN CONTENT AREA -->
+            </table>
+            <div class="footer">
+                <table border="0" cellpadding="0" cellspacing="0">
+                    <tr>
+                        <td class="content-block">
+                            <p><strong>Tento email dostávate, lebo ste sa prihlásili na odber upozornení na webe <a href="https://eia.cyklokoalicia.sk">Sleduj EIA</a>.</strong></p>
+                            <p><a href="{{ $project->unsubscribelinkloc }}" title="Zrušenie odberu upozornení pre danú lokalitu">Odhlásenie z odberu upozornení pre túto lokalitu</a>.</p>
```

Comment:
Pride mi haluzne, ze pri kazdom projekte je separatna unsubscribe linka. Chapem, ze je to preto, ze v jednom maile mozu prist projekty z roznych watcherov, ale je to divne. Zatial to teda spravme tak, ako si to spravil, a po nasadeni sa opytam Dana, ci chcu nejak zjednotit aj tie unsubscribe linky. Napriklad aspon pridat na koniec mailu vseobecnu linku unsubscribe vsetko.

Reply:
IMHO správne by to malo byť tak, že jedna emailová adresa má 1 unsubscribe link, ktorý ale otvorí nejakú page kde budú vylistované jednotlivé watchers a umožní buď unsubscribnúť sa úplne alebo z konkrétnych. nakoľko to ale je zatiaľ určite out-of-scope tak som to zatiaľ urobil takto